spinta/types/config.py

```diff
@@ -1,42 +1,38 @@
 from spinta.utils.imports import importstr
 from spinta.commands import load
-from spinta.components import Context, Config, BackendConfig
+from spinta.components import Context
+from spinta.config import Config
+from spinta import components
 
 
 @load.register()
-def load(context: Context, config: Config, data: dict) -> Config:
+def load(context: Context, config: components.Config, c: Config) -> Config:
```

Comment:
We have `components.Config` and `config.Config`, this will get confusing very quickly.

As I am not very knowledgable of how `components` module works and is used - I am not able to propose meaningful solution to this confusing naming.

Reply:
Yes, probably name should be changed. Maybe `spinta.config.Config` could be renamed to `RawConfig`.

Reply:
Maybe `AppConfig` sounds better? Though does it fit the semantics properly?

Reply:
`spinta.config.Config` is just a small wrapper on top of raw config dicts, env files and env vars.

`spinta.components.Config` uses `spinta.config.Config` to set attributes to a config objects that will be used in all other places in app. So this on sounds more like an `AppConfig`, and the other one like `RawConfig`.